benchmark.py

```diff
@@ -44,8 +36,27 @@
 parser.add_argument('--data-parallel', action = 'store_true')
 parser.add_argument('--backward', action = 'store_true')
 parser.add_argument('--input-time-dim-multiple', type = int, default = 128)
+
+parser.add_argument('--output', type=str)
+
 args = parser.parse_args()
 
+if args.save_cudnn_cublas_logs:
```

Comment:
Please report this issue to PyTorch (iiuc, setting these after `import torch` but before any torch usage still does not apply enable these logging). Since this is sort of unintuitive, you can do a feature request of enabling this logging in PyTorch API directly

Reply:
https://github.com/pytorch/pytorch/issues/58670